bedrock/mozorg/templates/mozorg/technology.html

```diff
@@ -206,7 +200,7 @@
 {% block email_form %}{% endblock %}
 
 {% block js %}
-  {% if not l10n_has_tag('emerging_tech_links') %}
+  {% if LANG.startswith('en-') or not l10n_has_tag('emerging_tech_links') %}
```

Comment:
There's a button to the play video in the fallback content for locales without the tag and we only need the JS for the video. I can add a comment to clarify.

However, the video is in English so I would be happy to remove that video button for all locales, regardless of tag. That would mean the fallback content would have no CTA at all for those few remaining locales but maybe we can live with that?